lib/classes/Variables.js

```diff
@@ -55,6 +55,7 @@ class Variables {
     this.cfRefSyntax = RegExp(/^(?:\${)?cf(\.[a-zA-Z0-9-]+)?:/g);
     this.s3RefSyntax = RegExp(/^(?:\${)?s3:(.+?)\/(.+)$/);
     this.ssmRefSyntax = RegExp(/^(?:\${)?ssm:([a-zA-Z0-9_.\-/]+)[~]?(true|false|split)?/);
+    this.toBoolRefSyntax = RegExp(/^(?:\${)?toBool:([a-zA-Z0-9_.\-/]+)/);
```

Comment:
I can change this to `${toBool(value)}` if that is preferred over `${toBool:value}`.